src/yield.js

```diff
@@ -15,117 +15,16 @@
  */
 
 /**
- * Returns a promise that is resolved in a new task. The resulting promise is
- * rejected if the associated signal is aborted. This uses scheduler.postTask()
- * to schedule continuations.
+ * Returns a promise that is resolved in a new task. This uses
+ * scheduler.postTask() to schedule continuations.
```

Comment:
SGTM, but now I'm wondering if we should tighten up src/polyfill.js? Like, should we handle the case where self.scheduler is defined but not scheduler.postTask(), which I guess would be trying to use the polyfill but there's already scheduler global?

Reply:
It might make sense? If a page is in that situation they probably have a lot more to worry about (why is there something already on `scheduler` but they also want this polyfill), but you never know, and if the polyfilling doesn't get too gross maybe that's ok to support them.

Could `yield` and `postTask` be bound to the scheduler instance and then added to the already-existing global?

If it does require too much work to cover all the unlikely possible permutations, the library could also expose a separate non-polyfilling entry point to import like any other library (though then they'd have to handle how to share a single scheduler instance throughout their code).

Reply:
> It might make sense? If a page is in that situation they probably have a lot more to worry about (why is there something already on `scheduler` but they also want this polyfill)...

True, and I'd assume the native version won't work for them in that case anyway. Maybe it's more trouble than it's worth. For now I've just added the fileoverview comment and documented the assumption.